testutil/promrated/metrics.go

```diff
@@ -9,44 +9,7 @@ import (
 )
 
 var (
-	validatorLabels = []string{"pubkey_full", "cluster_name", "cluster_hash", "cluster_network"}
-	networkLabels   = []string{"cluster_network"}
-
-	uptime = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "promrated",
-		Name:      "validator_uptime",
-		Help:      "Uptime of a validation key.",
-	}, validatorLabels)
-
-	correctness = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "promrated",
-		Name:      "validator_correctness",
-		Help:      "Average correctness of a validation key.",
-	}, validatorLabels)
-
-	inclusionDelay = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "promrated",
-		Name:      "validator_inclusion_delay",
-		Help:      "Average inclusion delay of a validation key.",
-	}, validatorLabels)
-
-	attester = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "promrated",
-		Name:      "validator_attester_effectiveness",
-		Help:      "Attester effectiveness of a validation key.",
-	}, validatorLabels)
-
-	proposer = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "promrated",
-		Name:      "validator_proposer_effectiveness",
-		Help:      "Proposer effectiveness of a validation key.",
-	}, validatorLabels)
-
-	effectiveness = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: "promrated",
-		Name:      "validator_effectiveness",
-		Help:      "Effectiveness of a validation key.",
-	}, validatorLabels)
+	networkLabels = []string{"cluster_network", "node_operator"}
```

Comment:
These literals are repeated explicitly in the code below. I suggest to introduce string constants for these.